packages/tasks/src/model-libraries.ts

```diff
@@ -266,6 +266,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.describe_anything,
 		filter: false,
 	},
+	"depthanything-ac": {
+		prettyLabel: "DepthAnything-AC",
+		repoName: "DepthAnythingAC",
```

Comment:
```suggestion
		repoName: "DepthAnything-AC",
```

let's keep same as "prettyLabel"